aloscene/panoptic.py

```diff
@@ -0,0 +1,100 @@
+import torch
+import numpy as np
+import cv2
+
+from typing import Union
+from torch import Tensor
+
+import aloscene
+from aloscene.renderer import View
+from aloscene.labels import Labels
+
+
+class Panoptic(aloscene.tensors.AugmentedTensor):
+    """Binary mask for each label to represent panoptic segmentations
+    """
+
+    @staticmethod
+    def __new__(cls, x: Tensor, labels: Union[dict, Labels] = None, names=("N", "H", "W"), *args, **kwargs):
```

Comment:
We must probably describe what type of information `labels` is supposed to  store. Before to merge,I would also suggest to add the docstrings in this class.

Reply:
Panoptic was replaced by update of `mask` class